docs/welcome.mdx

```diff
@@ -18,4 +18,4 @@ Our efforts are focused on three verticals:
 
 You can interact with Flashbots on Discord and Github. We have dedicated channels on Discord for each of our efforts and welcome your contributions. Our work is open source and you can follow our progress in each Github repository of the Flashbots organization.
 
-[Our mission](https://medium.com/flashbots/frontrunning-the-mev-crisis-40629a613752) | [Discord](https://discord.gg/7hvTycdNcK) | [Writings Website](https://docs.flashbots.net) | [Github](https://github.com/flashbots/pm) | [Transparency Reports](https://medium.com/flashbots/tagged/transparency-report)
+[Mission](https://writings.flashbots.net/writings/frontrunning-mev-crisis) | [Discord](https://discord.gg/7hvTycdNcK) | [Blog](https://writings.flashbots.net) | [Github](https://github.com/flashbots/pm) | [Transparency Reports](https://writings.flashbots.net/writings/tags/transparency-report)
```

Comment:
This should be in a separate repository. For next time, please split changes in different branches. For this time, no worries.

Reply:
sounds good - sorry about that!